client/src/main/java/io/split/engine/matchers/GreaterThanOrEqualToSemverMatcher.java

```diff
@@ -0,0 +1,54 @@
+package io.split.engine.matchers;
+
+import io.split.engine.evaluator.EvaluationContext;
+
+import java.util.Map;
+
+public class GreaterThanOrEqualToSemverMatcher implements Matcher {
+
+    private final Semver _semVer;
+
+    public GreaterThanOrEqualToSemverMatcher(String semVer) {
+        _semVer = Semver.build(semVer);
+    }
+
+    @Override
+    public boolean match(Object matchValue, String bucketingKey, Map<String, Object> attributes, EvaluationContext evaluationContext) {
+        if (matchValue == null) {
```

Comment:
same here, check if `_semVer` is null in this if.

Reply:
done